### DISCUSSION TITLE
OSSEC Certificate Mismatch

### DISCUSSION THREAD

**Discussion by adampankow** [Security-Onion-Solutions/securityonion#4538] [Unsupported Versions]
In the SOC alerts panel I'm seeing this OSSEC error:

`enrolling host: transport error in enrollment: rpc error: code = Unavailable desc = all SubConns are in TransientFailure, latest connection error: connection error: desc = \\\"transport: authentication handshake failed: x509: certificate is valid for hostname, not hostname.fqdn.com`

This seems to stem from following the setup script prompts and only entering the hostname on one prompt, then entering the FQDN that will be utilized when accessing SOC on the other prompt (for cookie security, IIRC). What would be the best way to resolve this mismatch issue after the fact? Should the setup script be reworked to use FQDN when setting up OSSEC, perhaps?

Thanks!

**Comment by weslambert** [comment]
To be clear, what is the process, etc this is being generated from in the OSSEC/Wazuh alert?  I've not noticed this before, however, that doesn't mean there isn't something strange going on. 😃

**Reply by adampankow** [reply]
The alert gives a process name of `launcher`, it also mentions it's being called by `updater.go`, and the rule name being fired on is `Unknown problem somewhere in the system`, with the the OSSEC/Wazuh error mentioned in the original post appearing in the original `/var/log/syslog` file that Beats shipped from the node, if that gives you something to go off of. Let me know if there's any additional information that might help you understand the issue. Thanks!

**Reply by UMHB-InfoSec** [reply]
Incoming zombie thread!

Did anyone ever find a resolution for this?   I am having this exact issue and its firing off to the tune of ~160,000 x per day.

Not a huge deal, but it is mucking up my interface a bit and I would like to fix it.

As always, a huge thanks to the community and devs!

**Reply by dougburks** [reply]
@UMHB-InfoSec I suspect you could tune the Wazuh rule as described at https://docs.securityonion.net/en/2.3/wazuh.html#modify-existing-rules. If you have further questions or problems, please start a new discussion rather than replying to this old discussion.
